qiskit_braket_provider/providers/braket_backend.py

```diff
@@ -327,8 +329,21 @@ def run(self, run_input, **options):
             del options["meas_level"]
 
         verbatim = options.pop("verbatim", False)
-        gateset = self._get_gateset() if not verbatim else None
-        braket_circuits = [to_braket(circ, gateset, verbatim) for circ in circuits]
+        native = options.pop("native", False)
```

Comment:
Up to you whether you want to make this change, but it might be better to include these two arguments in the method signature instead of just popping options